docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -292,3 +322,12 @@ Components not decided yet at current phase are not included in these views.
 ### Key Scenario: tenant & DNS provisioning
 
 ![tenant & DNS provision](images/network-scenario-tenant-init.png)
+
+### Activity Flow: Pod - from creation to running
+
+![Pod-activity-diag](images/pod-activity-diagram.png)
```

Comment:
A couple of questions about the digram:  1) Should the last column mean agent, instead of scheduler?  2) From the digram it seems the API Server have knowledge of flat network, it's a little strange.  The API Server shouldn't care about the network provider type, right?

Reply:
1. yes it should be agent in last column;
2. putting flat type check in diagram is improper; I'll update the diagram shortly